src/test/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/XGBoostEvidenceFilterUnitTest.java

```diff
@@ -0,0 +1,427 @@
+package org.broadinstitute.hellbender.tools.spark.sv.evidence;
+
+import biz.k11i.xgboost.Predictor;
+import biz.k11i.xgboost.util.FVec;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import org.apache.spark.api.java.JavaDoubleRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection;
+import org.broadinstitute.hellbender.tools.spark.utils.IntHistogram;
+import org.testng.AssertJUnit;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class XGBoostEvidenceFilterUnitTest extends GATKBaseTest {
+    private static final String testAccuracyDataJsonFile = publicTestDir + "sv_classifier_test_data.json";
+    private static final String classifierModelFile = "/large/sv_evidence_classifier.bin";
+    private static final String resourceClassifierModelFile = "gatk-resources::" + classifierModelFile;
+    private static final String localClassifierModelFile
+            = new File(publicMainResourcesDir, classifierModelFile).getAbsolutePath();
+    private static final String testFeaturesJsonFile = publicTestDir + "sv_features_test_data.json";
+    private static final double probabilityTol = 2.0e-4;
+    private static final double featuresTol = 1.0e-6;
+
+    private final ClassifierAccuracyData classifierAccuracyData = new ClassifierAccuracyData(testAccuracyDataJsonFile);
```

Comment:
Can this be `static`?

Reply:
Yes. I made this and a few other fields static.